t/29_sdaf_inventory_tools.t

```diff
@@ -101,9 +119,21 @@ subtest '[prepare_ssh_config] ' => sub {
     ok($db_host_B_content =~ /identity_file/, 'DB host A: identity_file');
     ok($db_host_B_content =~ /proxy_jump deployer_jump/, 'DB host A: proxy_jump');
     ok($db_host_B_content =~ /strict_host_key_checking no/, 'DB host A: strict_host_key_checking');
+};
+
+subtest '[create_redirection_data] ' => sub {
+    my $mock = Test::MockModule->new('sles4sap::sap_deployment_automation_framework::inventory_tools', no_auto => 1);
+    $mock->redefine(record_info => sub { return; });
```

Comment:
```suggestion
    $mock->redefine(record_info => sub { note(join(' ', 'RECORD_INFO -->', @_)); });
```

Reply:
Thanks for noting this. I actually had to remove it as the functions don't even user record_info. 
I think I used record info for debugging but forgot to remove this redefine.